components/renku_data_services/message_queue/converters.py

```diff
@@ -13,6 +13,12 @@
 from renku_data_services.project import models as project_models
 from renku_data_services.users import models as user_models
 
+QUEUE_NAME: Final[str] = "search.sync"
```

Comment:
suggestion: I think we probably want to rename this. Because for now it is only the search but in the future it may be more services that use this same stream. So I think the name should be based on the producer and event types rather than the current consumer. I.e. something like `data_service.all_events`? Hopefully it is not too complicated to make this change now.

Reply:
No, that is no problem! I'll change it to your suggested name. I thought we decided to use this stream now only to sync data between data-services and search. But obviously, if other are intended to read these events, the name is not good! Perhaps we then should also mark the "reprovisioning events" so consumers can distinguish them from "live events".